README.md

```diff
@@ -121,9 +123,21 @@ The data flow is as follows:
 3. Server takes post, and passes it to Gravity Forms as POST request
 4. Gravity Forms gets data
 
-Point 3 can be managed in multiple ways, depending on your build.
+Point 3 can be managed in multiple ways, depending on your build. For added security make sure you follow the steps below to implement reCAPTCHA on your form, preventing bots and spam.
 
-### Adding the Lambda (for Netlify)
+<details>
```

Comment:
Could you change this so it follows the heading hierarchy and isn't hidden from the user. The docs should be able to be quickly glanced across without needing to open any toggled areas. Thanks!